test/RewardsStreamerMP.t.sol

```diff
@@ -1898,7 +2019,7 @@ contract LockTest is RewardsStreamerMPTest {
         );
 
         // lock for another year should fail as 4 years is the maximum of total lock time
-        vm.expectRevert(StakeMath.StakeMath__AbsoluteMaxMPOverflow.selector);
+        vm.expectRevert(IStakeManager.StakingManager__InvalidLockPeriod.selector);
```

Comment:
Before we run into an absolute max overflow error, the stakemanager already knows that we'd exceed the totalLockTime and therefore reverts itself